.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -50,16 +50,6 @@ jobs:
         env:
           CCF_AZURE_SUBSCRIPTION_ID: $(CCF_AZURE_SUBSCRIPTION_ID)
 
-      - script: |
```

Comment:
Why is this removed? It seems that this is a useful step to make sure the container is ready before proceeding?

Reply:
It was a suggestion from Amaury to only wait on it when we actually need it, so the changes in test_on_remote should now wait on the deployment just before running ctest

Reply:
Also the main reason I'm doing this is that I've seen some failures this morning where the pipeline doesn't wait long enough, the script returns but ACI Is still setting up SSH

Reply:
@jumaffre this isn't necessary until we get to test_on_remote "Test Connection to Deployed ACIs", which does the exact same thing. So we can use the time to wrap up, and do the set up for test_on_remote.

Now that I think about it, I do not understand why they are two separate jobs, since they seem to run on the same pool. Can we not just have them be one job? They can still be separate files/templates (see common.yml for example).

Reply:
Yeah I'll make a second PR to combine the jobs, and after chatting to Julien we agreed waiting for the commands that setup SSH should be done from the python infra to deploy